smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -57,6 +57,8 @@ public async Task VerifyModuleIsRunning(string name)
                         await Task.Delay(TimeSpan.FromSeconds(3), cts.Token);
 
                         string[] result = await Process.RunAsync("iotedge", "list", cts.Token);
+                        WriteToConsole("Output of iotedge list", result);
```

Comment:
Do you want to remove this debug output before checking in? It will print potentially print out a list every 3 seconds.